src/pages/workspace/WorkspaceOverviewPage.tsx

```diff
@@ -214,6 +214,7 @@ function WorkspaceOverviewPage({policyDraft, policy: policyProp, route}: Workspa
         if (isComingFromGlobalReimbursementsFlow) {
             setIsComingFromGlobalReimbursementsFlow(false);
             Navigation.goBack();
+            return;
```

Comment:
Not sure why calling goBack and then popToSidebar causes the console error, but it's definitely not expected to call them both. The code above was added in https://github.com/Expensify/App/issues/60509, where pressing back will navigate the user back to the workspace initial page instead of the bank account page. So, we use `goBack` here but missing the early return.